### DISCUSSION TITLE
Wysyłam z Allegro

### DISCUSSION THREAD

**Discussion by Eskim83** [allegro/allegro-api#7769] [Suggestions - rejected]
Niedawno w związku z wymaganym wprowadzeniem OneBox do oferty wprowadziliśmy obsługę "Wysyłam z Allegro". Zrobiłem to według instrukcji w której uwzględniona została stara metoda. Wszystko działa, ale okazało się, że wprowadziliście całkowicie nową metodę, która znacząco komplikuje proces wymuszając podawanie numeru ulicy, kwoty ubezpieczenia, pobrania itd. Wiem już, że nie zdecyduję się prawdopodobnie na włączenie pobrań, bo jest to ryzykowne, a kwotę ubezpieczenia będę musiał wyciągać z umów i jakoś parować.

To co potrzebuję to funkcjonalność "Wysyłam z Allegro", która jest w Allegro:
* zlecenie wysyłki na podstawie id zamówienia
* uzupełnienie danych dotyczących rozmiarów, wagi itp. (choć w rzeczywistości są tu zawsze te same wartości), opis na paczce z wewnętrznym numerem zamówienia..
* poprawienie błędnych danych (np. klient wpisał źle numer ulicy, ale to tak naprawdę powinno być obsłużone po stronie serwisu)
* dodatkowa metoda do zamówienia podjazdu, albo jeszcze lepiej - automatyczne zamówienie podjazdu na najbliższy termin

i to wszystko.. Skomplikowaliście niektóre procesy na tyle, że poważnie zastanawiałem się nad webscrapingiem już, który byłby prostszy i pewniejszy (paradoksalnie) niż niektóre polecenia w API.

Razem z tym postem prośba o przemyślenie kwestii uproszczeń dla innych metod również. Chcecie metod ogólnych nie powiązanych z serwisem - ok, ale twórzcie też metody do obsługi stricte serwisu Allegro.

**Comment by MartaNowaczyk** [comment]
@Eskim83 Wracam z informacją od zespołu odpowiedzialnego za WZA - zespół potwierdził, że nie mamy w planach wdrażania przedstawionych tutaj rozwiązań, dlatego sugestię oznaczam jako odrzuconą.

**Comment by lasoczczyk** [comment]
Szkoda - zaoszczędziło by to sporo czasu... i pozwoliło uniknąć jakichś pomyłek itd....
